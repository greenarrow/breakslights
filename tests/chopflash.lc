MA01
MR01
A00 L
A00 CFffffff
A00 CBff0000
A00 FB1
A00 FC08
A00 FD01
A00 S04 I1
R00 N00
MS1a
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
MT
