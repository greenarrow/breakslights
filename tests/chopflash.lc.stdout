00 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000
00 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000
00 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000
00 ffffff ff0000 ff0000 ff0000 ff0000 ffffff ffffff ff0000 ff0000 ff0000 ff0000 ffffff
00 ffffff ff0000 ff0000 ff0000 ff0000 ffffff ffffff ff0000 ff0000 ff0000 ff0000 ffffff
00 ffffff ff0000 ff0000 ff0000 ff0000 ffffff ffffff ff0000 ff0000 ff0000 ff0000 ffffff
00 ffffff ff0000 ff0000 ff0000 ff0000 ffffff ffffff ff0000 ff0000 ff0000 ff0000 ffffff
00 ffffff ffffff ff0000 ff0000 ffffff ffffff ffffff ffffff ff0000 ff0000 ffffff ffffff
00 ffffff ffffff ff0000 ff0000 ffffff ffffff ffffff ffffff ff0000 ff0000 ffffff ffffff
00 ffffff ffffff ff0000 ff0000 ffffff ffffff ffffff ffffff ff0000 ff0000 ffffff ffffff
00 ffffff ffffff ff0000 ff0000 ffffff ffffff ffffff ffffff ff0000 ff0000 ffffff ffffff
00 ffffff ffffff ffffff ffffff ffffff ffffff ffffff ffffff ffffff ffffff ffffff ffffff
00 ffffff ffffff ffffff ffffff ffffff ffffff ffffff ffffff ffffff ffffff ffffff ffffff
00 ffffff ffffff ffffff ffffff ffffff ffffff ffffff ffffff ffffff ffffff ffffff ffffff
00 ffffff ffffff ffffff ffffff ffffff ffffff ffffff ffffff ffffff ffffff ffffff ffffff
00 ffffff ffffff ff0000 ff0000 ffffff ffffff ffffff ffffff ff0000 ff0000 ffffff ffffff
00 ffffff ffffff ff0000 ff0000 ffffff ffffff ffffff ffffff ff0000 ff0000 ffffff ffffff
00 ffffff ffffff ff0000 ff0000 ffffff ffffff ffffff ffffff ff0000 ff0000 ffffff ffffff
00 ffffff ffffff ff0000 ff0000 ffffff ffffff ffffff ffffff ff0000 ff0000 ffffff ffffff
00 ffffff ff0000 ff0000 ff0000 ff0000 ffffff ffffff ff0000 ff0000 ff0000 ff0000 ffffff
00 ffffff ff0000 ff0000 ff0000 ff0000 ffffff ffffff ff0000 ff0000 ff0000 ff0000 ffffff
00 ffffff ff0000 ff0000 ff0000 ff0000 ffffff ffffff ff0000 ff0000 ff0000 ff0000 ffffff
00 ffffff ff0000 ff0000 ff0000 ff0000 ffffff ffffff ff0000 ff0000 ff0000 ff0000 ffffff
00 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000
00 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000 ff0000
00 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000
00 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000
00 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000
00 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000
00 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000 000000
